src/tools/create-require.js

```diff
@@ -0,0 +1,21 @@
+'use-strict';
+
+const { dependencies, devDependencies } = require('../../package.json');
+const { RequireModuleError } = require('../errors');
+
+const createRequire = () => module => {
+  const libs = Object.keys(dependencies).concat(Object.keys(devDependencies));
+  const allowedLibs = ['lodash'];
+  const disallowedModules = ['fs', 'child_process'];
```

Comment:
given that the point of lambda is to run user code safely, I don't think there's a compelling reason to blacklist any modules right now. worst case, their code doesn't work as expected and they should use CLI instead. especially because godzilla is aimed at less technical users, i'm not sure this gains us anything